src/components/WorkOrders/MobileWorkingWorkOrdersView.js

```diff
@@ -95,7 +95,12 @@ const MobileWorkingWorkOrdersView = () => {
               <ol className="lbh-list mobile-working-work-order-list">
                 {renderWorkOrderListItems(
                   currentUser.isOneJobAtATime
-                    ? inProgressWorkOrders.slice(0, 1)
+                    ? inProgressWorkOrders.sort((a, b => {
+                      const dateA = new Date(a.assignedStartTime);
+                      const dateB = new Date(b.assignedStartTime);
+                      // Get first job by assigned start time
+                      return dateA - dateB;
+                    }).slice(0, 1))
```

Comment:
if anyone can think of a more succinct way to to this with strings of dates I'd be interested to hear it

Reply:
Could do `return new Date(a.assignedStartTime) - new Date(b.assignedStartTime);` directly but I think as you have it now is more readable. Could consider pulling it into a function?

Reply:
I actually quite like this implementation, verbosity is usually easier to understand. However, it would be better in a function. That way, you get the best of both worlds

Reply:
Refactored in to a new method